emrichen/tags/include.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from .base import BaseTag
+from ..void import Void
+
+
+class Include(BaseTag):
+    def enrich(self, context):
+        from ..template import Template
+
+        include_path = os.path.join(os.path.dirname(context['__file__']), self.data)
+        with open(include_path) as include_file:
```

Comment:
I wonder if we should be worried about filesystem traversal vulnerabilities here. No one's silly enough to use an user-specified variable as an input path... right? Right?

Reply:
We could always `chroot(2)` to the root template directory 🤔

Reply:
I'd leave that as the caller's responsibility, tbh. Just a thought. :D

Reply:
I'm pretty sure `!Lookup` or `!Format` could be used for arbitrary Python code execution as well. We should emphasize in the documentation that emrichen templates are unsafe and should be considered equal to Python code, anyway. You wouldn't run Python code uploaded by an untrusted user, nor should you render untrusted templates.